### ISSUE TITLE
Export button shouldn't highlight in green

### ISSUE THREAD

**Issue by eddavies** [cassproject/cass-editor#189]
The export button gets highlighted in green just like the editable fields. Is this intentional?

![download highlight green](https://user-images.githubusercontent.com/23706850/38213443-03854032-3676-11e8-840b-f2b77c549f6f.PNG)

**Comment by Lomilar** [comment]
Can you find the .class that is causing this? I looked for it (noticed it as well) and couldn't.

**Comment by Lomilar** [comment]
Oh, it looks like it comes from .changedField

Wrong color... I don't know what color to use, something like a light sea green?
Also shouldn't affect that particular dropdown.

**Comment by miledivovic** [comment]
Yeah, it was doing any <input> under detailSlider, made it more specific now.

**Comment by Lomilar** [comment]
Made color lighter.
